src/Processors/Formats/Impl/ORCBlockOutputFormat.cpp

```diff
@@ -482,10 +486,12 @@ void ORCBlockOutputFormat::consume(Chunk chunk)
     /// The size of the batch must be no less than total amount of array elements.
     ORC_UNIQUE_PTR<orc::ColumnVectorBatch> batch = writer->createRowBatch(getMaxColumnSize(chunk));
     orc::StructVectorBatch & root = dynamic_cast<orc::StructVectorBatch &>(*batch);
+    auto columns = chunk.getColumns();
```

Comment:
Can we use `detachColumns` columns? Seems that chunk are not used anymore.
```suggestion
    auto columns = chunk.detachColumns();
```

Reply:
Yes, we can, thanks.